package.json

```diff
@@ -7,7 +7,8 @@
   "author": "",
   "license": "LGPL-2.1",
   "scripts": {
-    "build": "webpack"
+    "build": "webpack",
+    "eslint": "eslint --ext .jsx --ext .es6 src/"
```

Comment:
Can you please also add this

    "eslint:fix": "eslint --fix --ext .jsx --ext .es6 src/"

This is rather useful for fixing whitespace, semicolons, etc.

Also, can you please add `--ext .js` as well, just in case?